i18n/fr/docusaurus-plugin-content-docs/current/community/breakout-rooms.md

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
Do we need this here? Also, it's in English still.

Reply:
I don't know if we need it here. 
I guess I didn't change this file. I will do the changes and then push 1 more commit with all the changes needed.